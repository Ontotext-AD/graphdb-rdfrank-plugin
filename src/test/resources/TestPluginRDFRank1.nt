@prefix : <http://www.ontotext.com/owlim/rdfrank/test#> .

:a :p :b .
:a :p :c .
:b :p :c .
