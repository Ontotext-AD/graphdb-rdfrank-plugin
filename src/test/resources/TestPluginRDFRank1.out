http://www.w3.org/1999/02/22-rdf-syntax-ns#type 0.00
http://www.w3.org/1999/02/22-rdf-syntax-ns#Property 0.00
http://www.w3.org/2000/01/rdf-schema#Class 0.00
http://www.w3.org/2000/01/rdf-schema#Resource 0.00
http://www.w3.org/2000/01/rdf-schema#subPropertyOf 0.00
http://www.w3.org/2000/01/rdf-schema#subClassOf 0.00
http://www.w3.org/2000/01/rdf-schema#domain 0.00
http://www.w3.org/2000/01/rdf-schema#range 0.00
http://www.w3.org/2002/07/owl#sameAs 0.00
http://www.w3.org/2002/07/owl#SymmetricProperty 0.00
http://www.w3.org/2002/07/owl#TransitiveProperty 0.00
http://www.w3.org/2002/07/owl#equivalentProperty 0.00
http://www.w3.org/2002/07/owl#equivalentClass 0.00
http://www.w3.org/2000/01/rdf-schema#Datatype 0.00
http://www.w3.org/2000/01/rdf-schema#ContainerMembershipProperty 0.00
http://www.w3.org/2000/01/rdf-schema#Literal 0.00
http://www.w3.org/2000/01/rdf-schema#member 0.00
http://www.ontotext.com/owlim/rdfrank/test#a 0.00
http://www.ontotext.com/owlim/rdfrank/test#p 0.00
http://www.ontotext.com/owlim/rdfrank/test#b 0.26
http://www.ontotext.com/owlim/rdfrank/test#c 1.00
